src/_gettsim/taxes/lohnst.py

```diff
@@ -93,19 +93,155 @@ def _lohnsteuer_klasse5_6_basis_y(taxable_inc: float, eink_st_params: dict) -> f
     return out
 
 
+@dates_active(
+    start="2019-01-01",
+    change_name="vorsorg_kv_option_b",
```

Comment:
```suggestion
    change_name="vorsorge_krankenv_option_b",
```

And similar below. Make it private for now in case it's too long.